Solutions/Auth0/Data Connectors/Auth0_FunctionApp.json

```diff
@@ -121,15 +121,15 @@
         "version": "1.0.0",
         "kind": "dataConnector",
         "source": {
-            "kind": "community"
+            "kind": "solution"
```

Comment:
The kind should be **Microsoft** not solution, please check in existing solutions... Thanks!!